src/main/webapp/filesFragment.xhtml

```diff
@@ -216,7 +216,7 @@
         <p:column class="col-file-metadata">
             
             <ui:fragment rendered="#{fileMetadata.datasetVersion.released or ((fileMetadata.datasetVersion.deaccessioned or fileMetadata.datasetVersion.draft) 
-                                     and permissionServiceBean.on(DatasetPage.dataset).has('ViewUnpublishedDataset'))}">
+                                        and permissionsWrapper.canViewUnpublishedDataset(dvRequestService.getDataverseRequest(), DatasetPage.dataset) )}"> 
```

Comment:
Updating this render logic to use the wrapper was the major cause of performance improvements. I'm not convinced the fragment needs to have a rendered tag at all though, as the other rows in the data table column do not and I am unsure when we would ever hide this part of the file info.